filecoin-proofs/src/api/post.rs

```diff
@@ -32,6 +33,8 @@ pub struct PrivateReplicaInfo {
     aux: PersistentAux,
     /// Is this sector marked as a fault?
     is_fault: bool,
+    /// Contains sector-specific (e.g. merkle trees) assets
+    cache_dir: PathBuf,
```

Comment:
do we need this?

Reply:
@dignifiedquire - Don't you want to re-use the sector-specific merkle trees generated during `seal` when generating a PoSt?